resources/public/api/conf/1.0/examples/businesses/create_response.json

```
@@ -0,0 +1,20 @@
+{
+  "businessId": "XAIS12345678910",
+  "links": [
+    {
+      "href": "/individuals/self-assessment-test-support/business/ET009363D/XAIS12345678910",
+      "method": "DELETE",
+      "rel": "delete-business"
+    },
+    {
+      "href": "/individuals/business/details/ET009363D/list",
+      "method": "GET",
+      "rel": "self"
+    },
```

Up to you, but `rel: self` doesn't entirely make sense to me, even though that's what the tech spec says we should use.

Firstly, you're saying:
> The rel will be self where the action is retrieval of the same resource.

And "list" doesn't retrieve the same resource. It retrieves all of them.

Secondly, it creates an ambiguous rel (confused with the more logical `rel: self` of retrieval). This prevents automatic parsing of related actions. I'm not suggesting changing the `rel` of retrieval. That one is what it should be.

I'll leave the decision to you, because I don't know what our use case for HATEOAS is, and if vendor-side automation is really what we are after. Plus with our current deadline, something imperfect here is better than nothing.

----

What it would look like: (plus changing the tech spec)
```suggestion
    {
      "href": "/individuals/business/details/ET009363D/list",
      "method": "GET",
      "rel": "list-all-businesses"
    },
```

---

The more I think about it, the more work this is... We'll sacrifice a lot of our throughput to address this because it impacts other APIs.